packages/nextra-theme-docs/src/components/breadcrumb.tsx

```diff
@@ -42,7 +43,7 @@ export const Breadcrumb: FC<{
                 href &&
                   'x:focus-visible:nextra-focus x:ring-inset x:hover:text-gray-900 x:dark:hover:text-gray-100'
               )}
-              title={item.title}
+              title={getLastStringChild(item.title)}
```

Comment:
```suggestion
              title={typeof item.title === 'string' ? item.title : undefined}
```
what do you think about this approach?

Reply:
yes, this will be another solution, but it will produce different results depending if you are using icons or not... I assumed that undefined makes that tooltip never appears, makes that final a or span has no title attribute, I'm right?

Reply:
You prefer this simple solution to less changes, right? In that way we can remove the added recursive function... as you want... I'd like the two solutions if I can solve not seeing "[object Object]" that is very annoying right now :)

Reply:
I would prefer a simpler solution for this edge case

Reply:
Nice. let me change in that way...

Reply:
Works as expected... with icon no title attribute otherwise as current live version. Good night!